bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -248,9 +248,9 @@ spec:
                 if [[ "${K8S_NODE_IP}" == "${CLUSTER_INITIATOR_IP}" ]]; then
                   echo "$(date -Iseconds) - nbdb - postStart - waiting for master to be selected"
 
-                  # set the connection and disable inactivity probe
+                  # set the connection and inactivity probe
                   retries=0
-                  while ! ovn-nbctl --no-leader-only -t 5 set-connection pssl:{{.OVN_NB_PORT}}{{.LISTEN_DUAL_STACK}} -- set connection . inactivity_probe=60000; do
+                  while ! ovn-nbctl --no-leader-only -t 5 set-connection pssl:{{.OVN_NB_PORT}}{{.LISTEN_DUAL_STACK}} -- set connection . inactivity_probe={{.OVN_NB_INACTIVITY_PROBE}}; do
```

Comment:
that variable is not defined? `/s/{{.OVN_NB_INACTIVITY_PROBE}}/{{.OVN_CONTROLLER_INACTIVITY_PROBE}}`

Reply:
My bad. I am adding an INACTIVITY_PROBE for NB, but failed to add it to CNO manifest file, thus causing this confusion. I have revised that on the latest commit.